doc/website/sidebars.js

```diff
@@ -43,6 +43,7 @@ module.exports = {
             label: 'Go API',
             items: [
                 {type: 'doc', id: 'go-api/intro', label: 'Introduction'},
+                {type: 'doc', id: 'go-api/inspection', label: 'Inspecting Schemas'},
```

Comment:
```suggestion
                {type: 'doc', id: 'go-api/inspect', label: 'Inspecting Schemas'},
```
maybe inspect instead